openstack/intel-worker/playbooks/roles/virtual_machine_create/templates/resolv.conf.j2

```diff
@@ -0,0 +1,2 @@
+search {{ virtual_machine_create_domain_name }}
+nameserver {{ virtual_machine_create_bastion_ip }}
```

Comment:
```suggestion
search {{ domain_name }}
nameserver {{ bastion_ip }}
```

Reply:
Ansible lint expects to have a role name as a prefix for the variable names .